### ISSUE TITLE
rbx required libedit-dev

### ISSUE THREAD

**Issue by digininja** [wayneeseguin/rvm#3269]
This is the tail of an error I got trying to install rbx 2.4.1:
```
/usr/local/rvm/rubies/ruby-2.2.0/lib/ruby/2.2.0/rake/application.rb:150:in `invoke_task'
/usr/local/rvm/rubies/ruby-2.2.0/lib/ruby/2.2.0/rake/application.rb:106:in `block (2 levels) in top_level'
/usr/local/rvm/rubies/ruby-2.2.0/lib/ruby/2.2.0/rake/application.rb:106:in `each'
/usr/local/rvm/rubies/ruby-2.2.0/lib/ruby/2.2.0/rake/application.rb:106:in `block in top_level'
/usr/local/rvm/rubies/ruby-2.2.0/lib/ruby/2.2.0/rake/application.rb:115:in `run_with_threads'
/usr/local/rvm/rubies/ruby-2.2.0/lib/ruby/2.2.0/rake/application.rb:100:in `top_level'
/usr/local/rvm/rubies/ruby-2.2.0/lib/ruby/2.2.0/rake/application.rb:78:in `block in run'
/usr/local/rvm/rubies/ruby-2.2.0/lib/ruby/2.2.0/rake/application.rb:176:in `standard_exception_handling'
/usr/local/rvm/rubies/ruby-2.2.0/lib/ruby/2.2.0/rake/application.rb:75:in `run'
/usr/local/rvm/rubies/ruby-2.2.0/bin/rake:33:in `<main>'
/usr/local/rvm/gems/ruby-2.2.0@rubinius/bin/ruby_executable_hooks:15:in `eval'
/usr/local/rvm/gems/ruby-2.2.0@rubinius/bin/ruby_executable_hooks:15:in `<main>'
Tasks: TOP => install => build:build => vm/vm
```

After quite a bit of searching for what that error meant I thought to look at the rake.log file and scroll back up to look for other errors. Thats where I found this:

```
Build time: 240.379790573 seconds
1: LD vm/vm
/usr/bin/ld: cannot find -ledit
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

Which says libedit-dev is missing. I installed that and the build worked fine.

Not sure if rvm is able to check for dependencies, maybe the rbx configuration script should do it and bail out a lot earlier, but thought I'd add this here just in case anyone else gets the error and can't work out what to do, hopefully it will save you some time.

**Comment by mpapis** [comment]
very important question, what system is it? maybe show `rvm info system`

**Comment by rys** [comment]
Fixed by #3308
